_gsocproposals/2019/proposal_AthenaMTResourceMonitoring.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Multi-thread safe resource monitoring infrastructure for the ATLAS experiment
+layout: gsoc_proposal
+project: ATLAS
+year: 2019
+organization:
+  - UCIrvine
+  - UOslo
+  - USheffield
```

Comment:
This organization does not exist, could you please also add a description together with a logo?

Reply:
Done.